### ISSUE TITLE
call for ground truth cluster label of amazon, dblp, lawyers datasets

### ISSUE THREAD

**Issue by dawnranger** [abojchevski/paican#1]
Thanks for the great work!  I'm trying to reproduce the experiments. But I can't find the ground truth cluster label(a.k.a. z.npy) of `amazon`, `dblp` and `lawyer` in the data folder. Since **NMI** is used to evaluate the results of PAICAN, it would be great help if you kindly release these information. Thanks!

**Comment by abojchevski** [comment]
For the `amazon` dataset we created there is no ground-truth cluster labels. That was the reason why we evaluated that dataset with a case study.

Similarly, the `dblp` dataset we used in this paper did not have any ground-truth cluster labels and we therefore again evaluated it with a case study. Note, that there are other datasets that are also called `dblp` which do include ground-truth class labels (e.g. typically they are used in node embedding papers). However, when we wrote the paper we were not aware of them so we used a version of the `dblp` dataset without ground-truth labels.

The `lawyer` dataset does have ground-truth labels. However, I can see that the currently uploaded `lawyers/A.mtx` and `lawyers/X.mtx` on the repository are incorrect (they are mistakenly copied from `cora`). I will locate the correct `A.mtx` and `X.mtx`, as well as `z.npy` and upload them all to the repository.
